HTML - 38/memorizeTheOrderOfRedCards.js

```diff
@@ -0,0 +1,75 @@
+let userCombination = [];
+let combination = randomArray();
+
+function randomArray() {
+    let a=[];
+    for (let i=0;i<30;++i) {
+        a[i]=i;
+    }
+    function shuffle(array) {
+        let tmp, current, top = array.length;
+        if(top) while(--top) {
+            current = Math.floor(Math.random() * (top + 1));
+            tmp = array[current];
+            array[current] = array[top];
+            array[top] = tmp;
+        }
+        return array;
+    }
+    a = shuffle(a);
+    let arr = [a[1], a[2], a[3], a[4]];

```

Comment:
There is slice method.
Using 4 elements here is sort of magic number, why are you returning 4 elements, that need to be in variable, after using slice you will see where to put that number.

Reply:
I used 4 because I needed to displayed 4 random cards to user. How do I use a variable with it?